core/iwasm/libraries/libc-wasi/libc_wasi_wrapper.c

```diff
@@ -63,6 +63,12 @@ typedef struct WASIContext {
 wasi_ctx_t
 wasm_runtime_get_wasi_ctx(wasm_module_inst_t module_inst);
 
+static inline size_t
+min(size_t a, size_t b)
```

Comment:
we already have the same function defined in `posix.c`. Maybe worth moving it to `bh_common`?

Reply:
It is a simple sentence and the function is defined as inline, my suggestion is just keep it here.